README.md

```diff
@@ -35,6 +35,27 @@ $ docker build -t my-nodejs-app .
 $ docker run -it --rm --name my-running-app my-nodejs-app
 ```
 
+If you prefer Docker Compose:
+
+```yaml
+version: "2"
+services:
+  node:
+    image: "node:4-onbuild"
```

Comment:
You should not use the `-onbuild` tag for this use case, since the image is not built when run through Docker Compose. The `-onbuild` variant should only be used in `FROM` when you are building a `Dockerfile`.